src/meta-srv/src/metasrv.rs

```diff
@@ -175,6 +177,8 @@ pub struct MetaSrv {
     mailbox: MailboxRef,
     ddl_manager: DdlManagerRef,
     table_metadata_manager: TableMetadataManagerRef,
+    #[cfg(feature = "version-report")]
+    version_reporter_task: Arc<VersionReportTask>,
```

Comment:
Could we provide a no-op version report task? If the `version-report` feature is disabled, it just does nothing.

Then we don't need to put the feature everywhere.